openlibrary/plugins/openlibrary/js/add-book.js

```diff
@@ -0,0 +1,14 @@
+export function initAddBookImport () {
+    window.q.push(function() {
+        $('.list-books a').click(function() {
+            var li = $(this).parents('li:first');
+            $('input#work').val('/works/' + $(li.attr('id')));
+            $('form#addbook').submit();
+        });
+    
```

Comment:
```suggestion

```

Reply:
Okay, that makes sense. I just deleted that space.